tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -1057,12 +1057,13 @@ def _return_value(obj, eng):
     expected_persistent_data_first_run = {
         'one': {'id': 1},
         'two': {'id': 2}
-    }
+    }.viewitems()
+
     expected_persistent_data_second_run = {
         'one': {'id': 1},
         'two': {'id': 2},
         'three': {'id': 43},
-    }
+    }.viewitems()
```

Comment:
```six.viewitems``?

Reply:
Do we want to make workflows and workflows tests ready for python3?